velox/buffer/StringViewBufferHolder.h

```diff
@@ -72,6 +72,13 @@ class StringViewBufferHolder {
     return stringBuffers_;
   }
 
+  /// Add a buffer to the list of buffers. This is used to allow bulk addition
+  /// of values with fewer overall buffers vs adding a value at a time via
+  /// getOwnedValue
```

Comment:
nit: Is `inBuffer` supposed to be owned by `pool_`? If so, could you specify this in the comment and add a `VELOX_CHECK_EQ(inBuffer->pool(), pool_)` in the method?